### ISSUE TITLE
Title not formatted

### ISSUE THREAD

**Issue by minchinweb** [yyoon/Journaley#115]
The title of the entry (i.e. the first line) is not being formatted. I guess whether or not this is an issue, you can decide, but the behavior is inconsistent with the DayOne app (DayOne formats the title with H1 formatting).

Example entry:

``` markdown
Second Test Entry

Some text

# H1

## H2

### H3

#### H4

##### H5

###### H6

More text
```

As rendered by Journaley 2.1:

![image](https://cloud.githubusercontent.com/assets/1548809/10295825/a87f9c2c-6b80-11e5-8715-9bc3de7da913.png)

As rendered by DayOne (on the iPad):

![2015-10-05 16 47 00](https://cloud.githubusercontent.com/assets/1548809/10295888/1f4b724a-6b81-11e5-8337-5747abd3c6fe.png)

**Comment by sguergachi** [comment]
I don't believe this is consistent with markdown, but your right it's something we should consider. Right now we render the title in the preview of the entry list box so maybe this is a behavior we should implement. It would require additional post processing to find the titles.

**Comment by sguergachi** [comment]
This might require the same work as https://github.com/yyoon/Journaley/issues/65
